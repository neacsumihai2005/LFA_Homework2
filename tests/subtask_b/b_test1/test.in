5
1 2 3 4 5
10
1 2 a
1 3 b
3 2 a
2 2 a
3 3 b
2 4 b
4 2 a
4 5 b
5 2 a
5 3 b
1
1
5
