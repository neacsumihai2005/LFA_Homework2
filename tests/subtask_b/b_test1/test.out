1 -> q0
2 -> q1
3 -> q2
4 -> q3
5 -> q4

{q0, q2}: ({q1}, a) ({q0, q2}, b) 
{q1}: ({q1}, a) ({q3}, b) 
{q3}: ({q1}, a) ({q4}, b) 
{q4}: ({q1}, a) ({q0, q2}, b) 
